package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -210,23 +253,150 @@ class WatsonCrickDist(NucPairDist):
     """
 
     def __init__(self, strand1: List[Residue], strand2: List[Residue],
-                 n1_name: str = 'N1', n3_name: str = "N3",
-                 g_name: str = 'G', a_name: str = 'A', u_name: str = 'U',
-                 t_name: str = 'T', c_name: str = 'C',
+                 nitrogen1_name: str = 'N1', nitrogen3_name: str = "N3",
+                 guanine_name: str = 'G', adenine_name: str = 'A', uracil_name: str = 'U',
+                 tymine_name: str = 'T', cytosine_name: str = 'C',
```

Comment:
See comment above — if you want to rename you'll also have to accept the old kwargs (possibly via `**kwargs`) for at least one minor release cycle.

Reply:
I just reverted to the old kwargs